src/Microsoft.AspNet.Identity/SignInResult.cs

```diff
@@ -40,74 +32,62 @@ public class SignInResult
         ///     Static success result
         /// </summary>
         /// <returns></returns>
-        public static SignInResult Success
-        {
-            get { return _success; }
-        }
+        public static SignInResult Success => new SignInResult { Succeeded = true };
 
         /// <summary>
         ///     Static failure result
         /// </summary>
         /// <returns></returns>
-        public static SignInResult Failed
-        {
-            get { return _failed; }
-        }
+        public static SignInResult Failed => new SignInResult();
 
         /// <summary>
         ///     Static locked out result
         /// </summary>
         /// <returns></returns>
-        public static SignInResult LockedOut
-        {
-            get { return _lockedOut; }
-        }
+        public static SignInResult LockedOut => new SignInResult { IsLockedOut = true };
 
         /// <summary>
         ///     Static not allowed result
         /// </summary>
         /// <returns></returns>
-        public static SignInResult NotAllowed
-        {
-            get { return _notAllowed; }
-        }
+        public static SignInResult NotAllowed => new SignInResult { IsNotAllowed = true };
 
         /// <summary>
         ///     Static two factor required result
         /// </summary>
         /// <returns></returns>
-        public static SignInResult TwoFactorRequired
-        {
-            get { return _twoFactorRequired; }
-        }
+        public static SignInResult TwoFactorRequired => new SignInResult { RequiresTwoFactor = true };
 
         /// <summary>
-        ///     Log result based on properties
+        ///     Returns string implemenation of the result. Provides string value for the assigned property
         /// </summary>
-        /// <param name="logger"></param>
-        /// <param name="message"></param>
-        public virtual void Log(ILogger logger, string message)
+        /// <returns></returns>
+        public override string ToString()
```

Comment:
Me and @divega had this chat and the consensus is that this is a behavior similar to enum so in that scenario it makes sense to hard code it.